### PR TITLE
CM: Display current workflow stage in the edit-view

### PR DESCRIPTION
**Pull request by gu-stav**
### What does it do?

This PR allows EE mode to render a different information element in the CM edit view. In EE mode it will contain the current workflow stage, that is selected for an entity.

### Why is it needed?

Allows users to see (not to change yet) the current workflow stage of an entity.

### CODE REVIEW

File: packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js [hunk 1]
```diff
@@ -28,22 +30,52 @@ export function useReviewWorkflows(workflowId) {
...
     return workflowUpdateMutation.mutateAsync({ workflowId, stages });
   }
 
+  function setStageForEntity(entityId, stageId) {
+    return entityStageMutation.mutateAsync({ entityId, stageId });
+  }
+
   function refetchWorkflow() {
     client.refetchQueries(workflowQueryKey);
   }
```
Comment (joshuaellis) [inline packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js:54]: Are you sure these don't need to be used in effects and therefore would benefit from being wrapped in `useCallback`? 🤔  just an API thought.

**Review by joshuaellis — APPROVED** [review]

File: packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js (hunk 1 above)
Comment (gu-stav) [reply packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js:54]: No, they don't have to. The can be called in a hook of the component that is consuming it though.

Comment (joshuaellis) [reply packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js:54]: > The can be called in a hook of the component that is consuming it though.

This is what I meant sorry, so if I use the hook somewhere else... But maybe that should be done on a case by case basis?

Comment (gu-stav) [reply packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js:54]: Yes ... I've made a mistake and shouldn't have included that in this PR already. There will be a separate one just for updating :)

**Pull request merged**